### PR TITLE
Azure - include basic sku in validation

### PR DESCRIPTION
**Pull request by stefangordon**
After finding an updated list of SKU's for plans - I realize this action will need to be modified to support the new "PC1" SKU - but I'm just going to add the missing B1, B2 and B3 in now since that will cover the vast majority of users, and fix the larger issue later.

### CODE REVIEW

**Comment by jdogg89** [comment]
Although this adds in the B tier, I'm also receiving a 'bad request' I mention in #3140 when trying to scale an s1 sku to and s2 sku. Are you able to successfully scale and app service plan in the previously supported sku's?

**Comment by stefangordon** [comment]
Ok so this has all possible SKU's now, and I no longer set two extra fields which the CLI team was not using.

File: tools/c7n_azure/c7n_azure/resources/appserviceplan.py [hunk 1]
```diff
@@ -56,14 +56,16 @@ class ResizePlan(AzureBaseAction):
...
                  count: 1
     """
 
     schema = utils.type_schema(
         'resize-plan',
         **{
-            'size': {'type': 'string', 'enum': ['F1', 'D1', 'S1', 'S2', 'S3', 'P1', 'P2', 'P3']},
+            'size': {'type': 'string', 'enum':
+                    ['F1', 'B1', 'B2', 'B3', 'D1', 'S1', 'S2', 'S3', 'P1', 'P2',
+                     'P3', 'P1V2', 'P2V2', 'P3v2', 'PC2', 'PC3', 'PC4']},
```
Comment (kapilt) [inline tools/c7n_azure/c7n_azure/resources/appserviceplan.py]: should this also have FREE, SHARED, BASIC, etc. else why is it in the sku name resolver if a user can't specify it?

Comment (stefangordon) [reply tools/c7n_azure/c7n_azure/resources/appserviceplan.py]: err - I was going back and forth between supporting those simplified names which the CLI supports, FREE==F1, SHARED==D1, etc and left it in-between.  Will clean that up.

**Pull request merged**